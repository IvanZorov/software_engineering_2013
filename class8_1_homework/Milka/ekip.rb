require 'csv'

array = Array.new()
argv = ARGV[1]

CSV.foreach(ARGV[0]) do |row|
	if argv == row[0]
		array << [row[1],row[2]]
	end
end

array.sort!{|a,b| a[1].to_i <=> b[1].to_i}

CSV.open("ekip_result.csv","w") do |csv|
	array.each do |element|
	csv << element
	end
end
