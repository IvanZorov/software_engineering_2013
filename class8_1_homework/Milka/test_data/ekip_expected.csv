tracksuit,42.50
sneakers,99.90
sweatshirt,99.90
sneakers,139.99
sweatshirt,140.90
sneakers,190.30
